src/components/CookieNotice/CookieNotice.tsx

```diff
@@ -0,0 +1,97 @@
+import "./CookieNotice.scss";
+import React from "react";
+import Backdrop from "@material-ui/core/Backdrop";
```

Comment:
@bitionaire and me are not really keen on using Googles material-ui. Maybe you could do it yourself?

Reply:
replaced material-ui with portal